maven-setup/src/main.ts

```diff
@@ -1,10 +1,19 @@
 import * as core from '@actions/core'
 import path from "path"
+import * as https from 'https'
+import * as fs from 'fs';
 
 import * as auth from '../../build-scan-shared/src/auth/auth'
 import * as errorHandler from '../../build-scan-shared/src/error'
 import * as input from '../../build-scan-shared/src/setup/input'
 import * as maven from '../../build-scan-shared/src/buildTool/maven'
+import {
```

Comment:
input functions are globally exported on line 8, you can access those with the `input` prefix

Reply:
Replaced